### PR TITLE
Helm - Add More Parameters

### PR DESCRIPTION
**Pull request by krasimirdermendzhiev**
#### Changelog

**New**

XSK chart:
* Add namespace value for deployment
* Add service account value 
* Add nameOverride, fullNameOverride value
* Add role collections value
* Add roleTemplate value
* Add scope value
* Add comments for parameters
* Add require kubernetes version

XSK-Kpack chart:
* Add namespace value
* Add require kubernetes version

**Changed**

XSK chart:
* Default role collection, roleTemplate, scope are deliver by values file.
* Changed README file

XSK-Kpack chart:
* Default builder images changed to Kyma
* Change README file

**Removed**

* xsk-kpack chart removed replicaCount

### CODE REVIEW

File: releng/helm-charts/xsk-kpack/templates/kpack.yaml [hunk 1]
```diff
@@ -21,14 +29,16 @@ apiVersion: kpack.io/v1alpha1
 kind: ClusterStore
 metadata:
   name: kneo-xsk-store
+  namespace: {{ .Values.create.namespace }}
 spec:
   sources:
-  - image: "{{ .Values.imageBuilder.buildpack }}:#{XSKVersion}#"
+  - image: "{{ .Values.imageBuilder.buildpack }}:0.11.0"
```
Comment (ThuF) [inline releng/helm-charts/xsk-kpack/templates/kpack.yaml:35]: Shouldn't it be `#{XSKVersion}#` instead of hardcoded version `0.11.0`?

**Review by ThuF — COMMENTED** [review]
@krasimirdermendzhiev one small comment about the version

File: releng/helm-charts/xsk-kpack/templates/kpack.yaml (hunk 1 above)
Comment (krasimirdermendzhiev) [reply releng/helm-charts/xsk-kpack/templates/kpack.yaml:35]: Yes I saw this. Thanks!

**Review by ThuF — APPROVED** [review]
+1

**Pull request merged**